db/migrate/20170531194154_add_multiple_redirect_uris.rb

```diff
@@ -0,0 +1,15 @@
+class AddMultipleRedirectUris < ActiveRecord::Migration
+  def up
+    add_column :service_providers, :redirect_uris, :json
```

Comment:
I was matching the style of the codebase already, there's a pre-existing JSON column but no other array columns:

```
identity-dashboard:master> git grep array -- db/migrate/
identity-dashboard:master> git grep json -- db/migrate/
db/migrate/20170117222117_add_attribute_bundle.rb:    add_column :service_providers, :attribute_bundle, :json
```